go.mod

```diff
@@ -113,4 +113,96 @@ require (
 	sigs.k8s.io/yaml v1.2.0
 )
 
-go 1.16
+require (
```

Comment:
Do we know why there's so much new stuff?

Reply:
Yeah, because the go version was bumped. This results in the pruning introduced in go 1.17 to be enabled:

> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the
> newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block
> from the block containing direct dependencies.

Ref https://go.dev/doc/go1.17

Reply:
*the go version in the go.mod file